redbot/cogs/mod/settings.py

```diff
@@ -365,3 +371,21 @@ async def defaultdays(self, ctx: commands.Context, days: int = 0):
                 days=days
             )
         )
+
+    @modset.command()
+    @commands.guild_only()
+    async def defaultduration(
+        self,
+        ctx: commands.Context,
+        duration: Optional[commands.TimedeltaConverter] = timedelta(days=0),
+    ):
+        """Set the default time to be used when a user is tempbanned."""
+        guild = ctx.guild
+        if not (timedelta(seconds=0) <= duration):
+            return await ctx.send(_("Invalid duration. Must be above 0."))
+        await self.config.guild(guild).default_tempban_duration.set(duration.total_seconds())
+        await ctx.send(
+            _("The default duration for tempbanning a user is now {days} ").format(
+                days=humanize_timedelta(timedelta=duration)
```

Comment:
`{days}` should be `{duration}` and sentence is missing a full stop:
```suggestion
            _("The default duration for tempbanning a user is now {duration}.").format(
                duration=humanize_timedelta(timedelta=duration)
```